pkg/eventprocessor/impl/webhook.go

```diff
@@ -76,14 +80,162 @@ func sendWebhookRequest(ctx context.Context, url string, body interface{}) error
 	return nil
 }
 
+type chain struct {
+	ID               int64
```

Comment:
Is the ID field necessary? I'm not seeing it being used and also it seems a bit redundant because the key of the map is the chain ID

Reply:
I think Name is also not being used